sparse_ho/algo/forward.py

```diff
@@ -37,7 +37,7 @@ def get_beta_jac_v(
 def get_beta_jac_iterdiff(
         X, y, log_alpha, model, mask0=None, dense0=None, jac0=None,
         max_iter=1000, tol=1e-3, compute_jac=True, return_all=False,
-        save_iterates=False, verbose=False, use_stop_crit=True):
+        save_iterates=False, verbose=False, use_stop_crit=True, Tgap=20):
```

Comment:
Tgap -> gap_freq to be consistent with celer

Reply:
my experience with this param for the lasso is that setting it to 10 is good and you don't need to change it, so wdyt about not exposing it ?

Reply:
It is not resolved. Call it gap_freq or don’t expose it. A variable with a capital letter like this is not ok

Reply:
It has been called gap_freq in the code :)